From 76293bb82fa2c4cc2c501f6711a2f637eee8f6b7 Mon Sep 17 00:00:00 2001
From: arathinx <ayyappanx.rathinam@intel.com>
Date: Mon, 19 Aug 2019 17:12:40 +0800
Subject: [PATCH]  CQM debugfs default mode is changed to 0400

 Giving the read only permission for users and none for group and others
---
 drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c b/drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c
index 601d66ff1076..3bbcd9fb8c8c 100644
--- a/drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c
+++ b/drivers/net/ethernet/lantiq/cqm/prx300/cqm_debugfs.c
@@ -1740,48 +1740,48 @@ int cqm_debugfs_init(struct cqm_ctrl *pctrl)
 	if (!pctrl->debugfs)
 		return -ENOMEM;
 
-	file = debugfs_create_file("cqmeqpc", 0644, pctrl->debugfs,
+	file = debugfs_create_file("cqmeqpc", 0400, pctrl->debugfs,
 				   pctrl, &cqm_eqpc_fops);
 	if (!file)
 		goto err;
 
-	file = debugfs_create_file("cqmdqpc", 0644, pctrl->debugfs,
+	file = debugfs_create_file("cqmdqpc", 0400, pctrl->debugfs,
 				   pctrl, &cqm_dqpc_fops);
 	if (!file)
 		goto err;
-	file = debugfs_create_file("cqmenq", 0644, pctrl->debugfs,
+	file = debugfs_create_file("cqmenq", 0400, pctrl->debugfs,
 				   pctrl, &cqm_enq_fops);
 	if (!file)
 		goto err;
 
-	file = debugfs_create_file("cqmdeq", 0644, pctrl->debugfs,
+	file = debugfs_create_file("cqmdeq", 0400, pctrl->debugfs,
 				   pctrl, &cqm_deq_fops);
 	if (!file)
 		goto err;
-	file = debugfs_create_file("cqmdma_desc", 0644,
+	file = debugfs_create_file("cqmdma_desc", 0400,
 				   pctrl->debugfs, pctrl, &cqm_dma_desc_fops);
 	if (!file)
 		goto err;
 
-	file = debugfs_create_file("cqmctrl", 0644, pctrl->debugfs,
+	file = debugfs_create_file("cqmctrl", 0400, pctrl->debugfs,
 				   pctrl, &cqm_ctrl_fops);
 	if (!file)
 		goto err;
 
-	file = debugfs_create_file("cqmls", 0644, pctrl->debugfs,
+	file = debugfs_create_file("cqmls", 0400, pctrl->debugfs,
 				   pctrl, &cqm_ls_fops);
 	if (!file)
 		goto err;
 
-	file = debugfs_create_file("ofsc", 0644, pctrl->debugfs,
+	file = debugfs_create_file("ofsc", 0400, pctrl->debugfs,
 				   pctrl, &cqm_ofsc_fops);
 	if (!file)
 		goto err;
-	file = debugfs_create_file("check_fsqm", 0644, pctrl->debugfs,
+	file = debugfs_create_file("check_fsqm", 0400, pctrl->debugfs,
 				   pctrl, &cqm_ofsq_fops);
 	if (!file)
 		goto err;
-	file = debugfs_create_file("pkt_count", 0644, pctrl->debugfs,
+	file = debugfs_create_file("pkt_count", 0400, pctrl->debugfs,
 				   pctrl, &cqm_dbg_cntrs_fops);
 	if (!file)
 		goto err;
